Shell/LineEditor.cpp

```diff
@@ -4,6 +4,8 @@
 #include <stdio.h>
 #include <unistd.h>
 
+#include <AK/StringBuilder.h>
```

Comment:
I don't see `StringBuilder` used anywhere?

Reply:
Whoops, forgot to remove this when I removed a function.